src/js/services/profileService.js

```diff
@@ -433,6 +433,39 @@ angular.module('copayApp.services')
       });
     };
 
+    root.checkLatestRelease = function(cb) {
+      var releaseURL = configService.getDefaults().release.url;
+
+      root.requestLatestRelease(releaseURL, function(err, release) {
+        if (err) return cb(err);
+
+        var actualVersion = window.version;
+        var latestVersion = release.data.tag_name;
+        var majorOfActualVersion = actualVersion.split('.')[0];
+        var majorOfLatestVersion = latestVersion.split('.')[0].slice(1);
+
+        if (majorOfLatestVersion <= majorOfActualVersion) return;
```

Comment:
the condition is wrong:

current | latest
1.0.0 | 1.1 -> should notify
1.5 | 2.0 -> should notify
1.1.1 | 1.1.2 -> should not notify